src/Sylius/Bundle/ShopBundle/Twig/Component/Checkout/Address/FormComponent.php

```diff
@@ -86,7 +100,14 @@ public function addressFieldUpdated(#[LiveArg] mixed $addressId, #[LiveArg] stri
         $newAddress['city'] = $address->getCity();
         $newAddress['postcode'] = $address->getPostcode();
 
-        $this->formValues[$field] = $newAddress;
+        $formData = new ArrayObject($newAddress);
```

Comment:
```suggestion
        $formData = new \ArrayObject($newAddress);
```
Use FQCN for base PHP classes.